### PR TITLE
Add user sessions list

### PR DESCRIPTION
**Pull request by adrian-gomez**

### CODE REVIEW

File: src/Page/User/Sessions/Sessions.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                        <ReactCountryFlag
+                          code={session.request_detail.country_code}
+                        />
+                      )}
+                  </td>
+                  <td>{session.authenticated_by}</td>
+                  <td>
+                    {session.request_detail &&
+                      session.request_detail.client_name}
+                    on&nbsp;
```
Comment (rickpr) [inline src/Page/User/Sessions/Sessions.js]:
```suggestion
                    &nbsp;on&nbsp;
```
image

File: src/Page/User/Sessions/Sessions.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+                    )}
+                    &nbsp;
+                    {session.request_detail &&
+                      session.request_detail.country_code && (
+                        <ReactCountryFlag
+                          code={session.request_detail.country_code}
+                        />
+                      )}
+                  </td>
+                  <td>{session.authenticated_by}</td>
```
Comment (rickpr) [inline src/Page/User/Sessions/Sessions.js]:
```suggestion
                  <td>{humanizeFormatter(session.authenticated_by)}</td>
```

File: package-lock.json [hunk 3]
```diff
@@ -5093,8 +5093,7 @@
         },
         "ansi-regex": {
           "version": "2.1.1",
-          "bundled": true,
-          "optional": true
```
Comment (rickpr) [inline package-lock.json]: What's up with this? I don't see any `package.json` changes.

File: src/Page/User/Sessions/Sessions.js (hunk 1 above)
Comment (adrian-gomez) [reply src/Page/User/Sessions/Sessions.js]: done!

File: src/Page/User/Sessions/Sessions.js (hunk 2 above)
Comment (adrian-gomez) [reply src/Page/User/Sessions/Sessions.js]: we do the same on provider portal I mostly want to keep consistent with that (when possible)

File: package-lock.json (hunk 3 above)
Comment (adrian-gomez) [reply package-lock.json]: not sure, I can undo them tho

File: src/Page/User/Sessions/Sessions.js (hunk 2 above)
Comment (rickpr) [reply src/Page/User/Sessions/Sessions.js]: Seems fine to me if that's the case, we can decide to do this later.

File: package-lock.json (hunk 3 above)
Comment (adrian-gomez) [reply package-lock.json]: not sure, I can undo them tho

**Pull request merged**